packages/adblocker/src/engine/engine.ts

```diff
@@ -486,14 +509,34 @@ export default class FilterEngine extends EventEmitter<
   /**
    * Update engine with new filters as well as optionally removed filters.
    */
-  public update({
-    newNetworkFilters = [],
-    newCosmeticFilters = [],
-    removedCosmeticFilters = [],
-    removedNetworkFilters = [],
-  }: Partial<IListDiff>): boolean {
+  public update(
+    {
+      newNetworkFilters = [],
+      newCosmeticFilters = [],
+      newPreprocessors = [],
+      removedCosmeticFilters = [],
+      removedNetworkFilters = [],
+      removedPreprocessors = [],
+    }: Partial<IListDiff>,
+    env = new Env(),
```

Comment:
Not sure if already inferred from the default value but we could add an explicit type:
```suggestion
    env: Env = new Env(),
```